lib/decidim/maintainers_toolbox/action_backporter.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require_relative "github_manager/querier"
+require_relative "github_manager/poster"
+require_relative "git_backport_manager"
+
+module Decidim
+  module MaintainersToolbox
+    class ActionBackporter
+      class InvalidMetadataError < StandardError; end
+
+      # @param token [String] token for GitHub authentication
+      # @param pull_request_id [String] the ID of the pull request that we want to backport
+      # @param exit_with_unstaged_changes [Boolean] wheter we should exit cowardly if there is any unstaged change
+      def initialize(token: , pull_request_id: ,exit_with_unstaged_changes: )
+        @token = token
+        @pull_request_id = pull_request_id
+        @exit_with_unstaged_changes = exit_with_unstaged_changes
+      end
+
+      def call
+        exit_with_errors("The requested PR #{pull_request_id} does not contain `type: fix`") unless pull_request_metadata[:labels].include?("type: fix")
+        exit_with_errors("The requested PR #{pull_request_id} is not merged") unless pull_request_metadata[:is_merged]
+        exit_with_errors("The requested PR #{pull_request_id} cannot be backported") if pull_request_metadata[:labels].include?("no-backport")
+
+        extract_versions.each do |version|
+          next if extract_backport_pull_request_for_version(related_issues, version)
+          system("decidim-backporter --github_token=#{token} --pull_request_id=#{pull_request_id} --version_number=#{version} --exit_with_unstaged_changes=#{exit_with_unstaged_changes} --with-console=false", exception: true)
+        rescue RuntimeError => e
+          puts e.message
+          create_backport_task(version)
+        end
+      end
+
+      private
+
+      attr_reader :token, :pull_request_id, :exit_with_unstaged_changes
+
+      def create_backport_task(version)
```

Comment:
I think it's a bit clearer
```suggestion
      def create_backport_issue(version)
```